### ISSUE TITLE
results.png

### ISSUE THREAD

**Issue by yinyuhangi** [ultralytics/yolov5#1462]
## ❔Question
![image](https://user-images.githubusercontent.com/46810994/99809102-c8d0d380-2b7c-11eb-9ea6-be086b67f548.png)
Thanks for your time. 
Are the Precision, Recall and mAP0.5 curves in the figure of training set or validation set?

**Comment by glenn-jocher** [comment]
All 4 metrics are always computed on the validation set 😃

**Comment by yinyuhangi** [comment]
ok, thanks.
